### ISSUE TITLE
Bad reconstruction on synthetic avatar

### ISSUE THREAD

**Issue by NytePlus** [GaussianCube/GaussianCube_Construction#7]
Hello, thanks for your excellent work! I have a question about avatar reconstruction. I used the same parameters as the paper: 300 multi view images with elevation angles between [-30 °, 60 °], l1_loss and ssim loss, and achieved poor reconstruction results with a training step size of 30k.

![Image](https://github.com/user-attachments/assets/34e91a6c-0f91-459c-bceb-5bb0dfbfb082)
I also tried lpips loss and l2 loss. The image quality has improved, but the 3D is inconsistent. (eyes)

![Image](https://github.com/user-attachments/assets/3305cbf6-b95f-4432-b160-cda17cb7aa76)

May I ask for your suggestions on parameters?

**Comment by NytePlus** [comment]
sorry, I made a mistake about camera matrix.
